android/src/main/java/com/rcttabview/RCTTabView.kt

```diff
@@ -140,4 +142,31 @@ class ReactBottomNavigationView(context: Context) : BottomNavigationView(context
     super.onDetachedFromWindow()
     isAnimating = false
   }
+
+  private fun updateTintColors() {
+    val activeColor = activeTintColor ?: return
+    val inactiveColor = inactiveTintColor ?: return
```

Comment:
What if someone only sets activeColor or inactiveColor? In that case this won't work.

Reply:
Either that or we have to set some default value of both colors as we can't make `ColorStateList` with null values

Maybe we should access material theme value and add it as default value WDYT?

Reply:
Updated ✅

Reply:
Looks good!